qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -1838,7 +1838,7 @@ def test_backlog_same_correlation_id_no_end(self):
                             trial,
                             model_name,
                             dtype,
-                            1001,
+                            1002,
```

Comment:
What is the reason for this change?

Reply:
I found that CORRID 1001 would exceed max_sequence_idle_microseconds while waiting for sending all the other sequences, and then got force-ended, which leads the scheduler to hang because it couldn't receive the enough number of inferences. I was thinking that maybe switching the order of the first two threads can prevent the sequence without an end marker from getting terminated.

Reply:
What is the `max_sequence_idle_microseconds` for the model? From this test's [description](https://github.com/triton-inference-server/server/blob/main/qa/L0_sequence_batcher/sequence_batcher_test.py#L1723-L1730), I would think that the timeout is quite long to test that "the new sequence must start without waiting for the timeout"

Reply:
It is also strange that changing the order helps, in this test the first two threads are started [back-to-back](https://github.com/triton-inference-server/server/blob/main/qa/L0_sequence_batcher/sequence_batcher_test.py#L1852-L1857), unless the Python's thread launch has significant overhead in Jetson.

Reply:
`max_sequence_idle_microseconds` is set to 5000000 based on model configuration. In normal cases, the new sequence does start without waiting for the timeout, but I found that this issue occurred in lots of pipelines since last week, and my guess would be that it took longer than we expected for the threads being launched. 
For example, I was checking "[test_backlog_fill_no_end](https://github.com/triton-inference-server/server/blob/main/qa/L0_sequence_batcher/sequence_batcher_test.py#L1536-L1546)" test case. Originally the delay between each thread is set to 2 seconds, and the order of sequences the server received should be CORRID 1001, 1002 ... 1006. But from the loggings I got, the order could be wrong sometimes. The server could actually receive 1005 before receiving 1004, and thus the scheduler got hang. So I changed the delay to 4 seconds, and seems like this happens lesser than before.

Reply:
That is 5 seconds, substracting the 2 seconds that the script waits for before launching 5th sequence, 3 seconds is a long time for sending requests. That looks like a regression that we want to resolve rather than just making the test less time sensitive. The test runs for all backends, does this timeout happen for all backends or only for a subset of backends?